e2e-tests/package.json

```diff
@@ -14,6 +14,8 @@
     "showcase-rbac-nightly": "playwright test --project=showcase-rbac",
     "showcase-k8s-ci-nightly": "playwright test --project=showcase-k8s",
     "showcase-rbac-k8s-ci-nightly": "playwright test --project=showcase-rbac-k8s",
+    "showcase-operator-nightly": "playwright test --project=showcase-operator",
```

Comment:
I believe we are expected to run all the tests that we use on Helm deployment on the operator as well. It makes sense to call `playwright test --project=showcase` and `playwright test --project=showcase-rbac` instead of adding a new project in config.

Reply:
I believe this is similar to k8s tests. Some of the tests are not yet adapted to work on Operator and until then we need to add them to `testIgnore` thus we need another project.  
Even after we achieve the parity of Operator and Helm, we can have some of the tests failing on one of them and we will have to temporarily disable it. Better to do that just for the Operator than skip the test for both.

Reply:
I am okay with the plan. Are there JIRA tickets just for tracking the skipped test cases, so we don't miss them?

Reply:
I opened this JIRA ticket for the operator https://issues.redhat.com/browse/RHIDP-5366.